Analysis/Tasks/PWGHF/taskDPlus.cxx

```diff
@@ -90,13 +99,71 @@ struct TaskDPlus {
       registry.fill(HIST("hd0Prong0"), candidate.impactParameter0());
       registry.fill(HIST("hd0Prong1"), candidate.impactParameter1());
       registry.fill(HIST("hd0Prong2"), candidate.impactParameter2());
-      //registry.fill(HIST("hSelectionStatus"), candidate.isSelDPlus());
     }
   }
 };
 
-WorkflowSpec defineDataProcessing(ConfigContext const&)
+/// D± analysis task for MC
+struct TaskDPlusMC {
+  HistogramRegistry registry{
+    "registry",
+    {{"hPtRecSig", "3-prong candidates (rec. matched);#it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
+     {"hPtRecBg", "3-prong candidates (rec. unmatched);#it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
+     {"hPtGen", "3-prong candidates (gen. matched);#it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
+     {"hPtGenSig", "3-prong candidates (rec. matched);#it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
+     {"hEtaRecSig", "3-prong candidates (rec. matched);#it{#eta};entries", {HistType::kTH1F, {{100, -2., 2.}}}},
+     {"hEtaRecBg", "3-prong candidates (rec. unmatched);#it{#eta};entries", {HistType::kTH1F, {{100, -2., 2.}}}},
+     {"hEtaGen", "3-prong candidates (gen. matched);#it{#eta};entries", {HistType::kTH1F, {{100, -2., 2.}}}}}};
```

Comment:
Adjust the titles for MC particle histograms. (See `taskD0.cxx`.)

Reply:
Done